### PR TITLE
refactor: modernize OrganisationSettingsPage

### LINKED ISSUE #5779: Migrate critical components: OrganisationSettingsPage
Critical component used for configuration.

**OrganisationSettingsPage**
- Class component
- 464 lines

## Objectives
Ranked by priority
- Migrate it to a Functional component (along with TS)
- Break it in small / modular and dedicated components
- Remove the Providers injections to use RTK

### PR DESCRIPTION
**Pull request by talissoncosta**
Thanks for submitting a PR! Please check the boxes below:

- [ ] I have added information to `docs/` if required so people know about the feature!
- [x] I have filled in the "Changes" section below?
- [x] I have filled in the "How did you test this code" section below?
- [x] I have used a [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) title for this Pull Request

### Overview
Migrated `OrganisationSettingsPage` from a 466-line JavaScript class component to a modern TypeScript functional component architecture, replacing legacy Flux patterns with RTK Query and decomposing the monolithic component into well-structured, reusable modules.

#### 1. **Migration to TypeScript**
- Converted from JavaScript class component to TypeScript functional component
- Added proper type definitions for all props, state, and API responses
- Full type safety throughout the component tree
- Created `UpdateOrganisationBody` type in `requests.ts`
- Added `OrganisationSettingsTab` type for tab configuration

#### 2. **RTK Query Integration**
- **Replaced legacy stores**: Eliminated `AccountStore` and `OrganisationProvider` dependencies
- **Added new endpoints** to `useOrganisation.ts`:
  - `getOrganisation` - Fetch single organisation with proper cache tags
  - `updateOrganisation` - Update organisation with optimistic updates
  - `deleteOrganisation` - Delete organisation with cache invalidation
- **Optimistic updates**: Organisation updates appear immediately in UI with automatic rollback on error
- **Automatic cache invalidation**: Changes propagate to all components using the same data

#### 3. **Custom Hooks for Reusable Logic**
- **`useUpdateOrganisationWithToast`**: Encapsulates update logic with toast notifications and error handling
- **`useDeleteOrganisationWithToast`**: Handles organisation deletion with:
  - Automatic calculation of next available organisation for navigation
  - Legacy store synchronization via `AppActions.refreshOrganisation()` for backward compatibility
  - Toast notifications for success/error states
  - Loading state management

#### 4. **Component Decomposition**
Broke down the monolithic 466-line component into focused, maintainable modules:

```
organisation-settings/
├── OrganisationSettingsPage.tsx (136 lines) - Main orchestrator
├── hooks/
│   ├── useUpdateOrganisationWithToast.ts - Update hook with toast
│   └── useDeleteOrganisationWithToast.ts - Delete hook with navigation
└── tabs/
    ├── general-tab/
    │   ├── index.tsx - GeneralTab orchestrator
    │   └── sections/
    │       ├── OrganisationInformation.tsx - Name update form
    │       ├── DeleteOrganisation.tsx - Deletion UI
    │       └── admin-settings/
    │           ├── Force2FASetting.tsx - 2FA toggle
    │           └── RestrictProjectCreationSetting.tsx - Project creation restriction
    ├── BillingTab.tsx - Billing & subscription management
    ├── LicensingTab.tsx - Enterprise licensing
    ├── CustomFieldsTab.tsx - Custom fields management
    ├── APIKeysTab.tsx - API keys management
    ├── WebhooksTab.tsx - Webhook configuration
    └── SAMLTab.tsx - SAML authentication
```

#### 5. **Modern React Patterns**
- **Hooks over HOCs**: Removed `withRouter` and `ConfigProvider` HOCs
- **useRouteContext**: Replaced `match.params` with context-based routing
- **useHistory**: Direct hook usage instead of prop drilling
- **Proper imports**: Uses `{ FC, ReactNode }` instead of `React.FC`
- **Functional components**: All components are now functional with hooks

#### 6. **Navigation Improvements**
- **Explicit navigation logic**: After organisation deletion, explicitly calculates next organisation
- **Smart routing**:
  - If user has other orgs → navigate to `/organisation/{nextOrgId}/projects`
  - If no orgs remaining → navigate to `/create` (create organisation page)
- **No reliance on global state**: Navigation based on RTK Query data, not AccountStore

#### 7. **Loading States & UX**
- **Delete button**: Shows "Deleting..." text and disables during operation
- **Update button**: Shows "Updating" text and disables during save
- **Page-level loading**: Displays centered loader during initial data fetch
- **Error states**: Shows clear error messages for failed operations

#### 8. **Backward Compatibility**
- **Maintains `AppActions.refreshOrganisation()`**: Ensures legacy components (navbar, permissions UI) stay in sync
- **Dual sync strategy**: RTK Query handles modern components, AppActions handles legacy components
- **Gradual migration path**: Allows incremental migration away from legacy stores

#### 9. **Tab Array Pattern**
Implemented clean, declarative tab configuration matching `ProjectSettingsPage`:
```typescript
const tabs: OrganisationSettingsTab[] = [
  {
    component: <GeneralTab organisation={organisation} />,
    isVisible: true,
    key: 'general',
    label: 'General',
  },
  // ... more tabs
].filter(({ isVisible }) => isVisible)
```

#### 10. **Permission Handling**
- **Direct API checks**: Permission checks from `organisation.role` instead of AccountStore
- **Feature flag integration**: Uses `Utils.getFlagsmithHasFeature()` for conditional features
- **Payment method checks**: Billing tab visibility based on `organisation.subscription.payment_method`

#### 11. **Cache Invalidation & Navigation Synchronization**
- **RTK Query cache tags pattern**: Using `{ id: 'LIST', type: 'Organisation' }` for automatic list invalidation
- **BreadcrumbSeparator migration**: Migrated to RTK Query so navbar reflects changes instantly without manual refresh
- **Mutation invalidation strategy**:
  - `updateOrganisation` invalidates `{ id: 'LIST', type: 'Organisation' }` and specific organisation cache
  - `deleteOrganisation` invalidates `{ id: 'LIST', type: 'Organisation' }` to refresh organisation dropdown
  - `createOrganisation` invalidates cache to show new organisation in nav immediately
- **Cross-component synchronization**: Changes in settings page automatically propagate to navbar, breadcrumbs, organisation selector, and all other components consuming organisation data
- **Eliminates manual refresh needs**: No need for page reloads or manual store updates - RTK Query handles all data synchronization

---

#### ✅ **Organisation Information Updates**
- [x] Update organisation name → verified name updates in navbar
- [x] Update with empty name → verified validation prevents submission
- [x] Update with very long name → verified it saves correctly
- [x] Cancel edit → verified changes are discarded
- [x] Check loading state → "Updating" text appears during save

#### ✅ **Admin Settings**
- [x] Toggle "Force 2FA" setting → verified it persists after page reload
- [x] Toggle "Restrict Project Creation" → verified setting saves correctly
- [x] Verify settings persist across page reloads
- [x] Check that non-admin users cannot access these settings

#### ✅ **Organisation Deletion**
- [x] Delete organisation with multiple orgs → navigates to next organisation's projects page
- [x] Delete organisation (last one) → navigates to `/create` (create organisation page)
- [x] Check confirmation modal appears before deletion
- [x] Verify loading state during deletion ("Deleting..." text)
- [x] Cancel deletion → modal closes without action
- [x] Verify deleted org disappears from navbar immediately

#### ✅ **Tab Navigation**
- [x] Switch between all tabs → all tabs load correctly
- [x] Billing tab visibility → hidden for AWS Marketplace customers
- [x] Licensing tab visibility → only visible in enterprise installations
- [x] Tab state persists in URL (`?tab=billing`)
- [x] Direct URL access to specific tab works correctly

#### ✅ **Permission Checks**
- [x] Admin user → can access all settings
- [x] Non-admin user → sees "You do not have permission" message
- [x] Org not found → error message displays
- [x] Network error → error message displays

#### ✅ **Loading States**
- [x] Initial page load → centered loader appears
- [x] Organisation update → button shows "Updating"
- [x] Organisation delete → button shows "Deleting..." and is disabled
- [x] Tab switches → content updates immediately (no loader needed, already cached)

#### ✅ **Backward Compatibility**
- [x] Navbar organisation list updates after changes
- [x] Legacy permissions UI components still function
- [x] Organisation selector in other components reflects changes
- [x] No console errors related to store updates

#### ✅ **Error Handling**
- [x] API error during load → error message displays
- [x] API error during update → toast notification appears
- [x] API error during delete → toast notification appears
- [x] Network timeout → appropriate error handling

#### ✅ **Edge Cases**
- [x] Organisation ID not in URL → handled gracefully
- [x] Invalid organisation ID → error message displays
- [x] Multiple rapid clicks on save → prevents double submission
- [x] Multiple rapid clicks on delete → prevents double submission
- [x] Browser back/forward buttons → navigation works correctly

### ✅ **E2E Test Coverage**
Added comprehensive E2E tests in `frontend/e2e/tests/organisation-test.ts`:

**Test 1: Edit Organisation Name and Verify Updates**
- [x] Update organisation name in settings
- [x] Navigate to projects page and verify name appears in navbar breadcrumb
- [x] Navigate back to settings and verify name persisted
- [x] Confirms cache invalidation works correctly across navigation

**Test 2: Create and Delete Organisation (Multiple Orgs Exist)**
- [x] Create new organisation and verify it appears in navbar
- [x] Delete the newly created organisation
- [x] Verify redirect to next available organisation's projects page
- [x] Verify deleted organisation no longer appears in navbar dropdown
- [x] Confirms proper cache updates and navigation logic

**Test 3: Cancel Organisation Deletion**
- [x] Create temporary organisation
- [x] Open delete modal and fill in confirmation
- [x] Cancel deletion by closing modal
- [x] Verify organisation still exists in navbar
- [x] Confirms cancellation flow prevents accidental data loss

**Test 4: Organisation Name Validation**
- [x] Clear organisation name field
- [x] Verify save button is disabled with empty name
- [x] Restore original name
- [x] Verify save button is enabled with valid name
- [x] Confirms proper form validation

**Note:** The edge case of deleting the last organisation (redirect to `/create`) is not covered by E2E tests, as it would require complex test isolation. This behavior is tested via manual testing and can be covered by unit tests of the delete hook logic.

Closes #5779

### CODE REVIEW

File: frontend/common/services/useOrganisation.ts [hunk 1]
```diff
@@ -15,6 +31,39 @@ export const organisationService = service
           url: `organisations/`,
         }),
       }),
+      updateOrganisation: builder.mutation<
+        Res['organisation'],
+        Req['updateOrganisation']
+      >({
+        invalidatesTags: (res) => [
+          { id: res?.id, type: 'Organisation' },
```
Comment (Zaimwa9) [inline frontend/common/services/useOrganisation.ts]: What happens if `res?.id` is undefined ?

File: frontend/common/types/requests.ts [hunk 2]
```diff
@@ -151,6 +158,9 @@ export type Req = {
     environments?: string
   }>
   getOrganisations: {}
+  getOrganisation: { id: string | number }
+  updateOrganisation: { id: string | number; body: UpdateOrganisationBody }
+  deleteOrganisation: { id: string | number }
```
Comment (Zaimwa9) [inline frontend/common/types/requests.ts]: I'd say let's commit to either string or number in these interfaces to avoid confusion when consuming them

File: frontend/web/components/pages/OrganisationSettingsPage.js [hunk 3]
```diff
@@ -1,466 +0,0 @@
...
-      'p-0',
-    )
-  }
-
-  onRemove = () => {
-    toast('Your organisation has been removed')
-    if (AccountStore.getOrganisation()) {
-      this.props.history.replace(Utils.getOrganisationHomePage())
-    } else {
-      this.props.history.replace('/create')
```
Comment (Zaimwa9) [inline frontend/web/components/pages/OrganisationSettingsPage.js:79]: Is there a reason why we changed the `/create` fallback? I haven't gone through this flow in a while but curious to have the rationale behind

File: frontend/web/components/pages/organisation-settings/tabs/general-tab/sections/admin-settings/Force2FASetting.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+export const Force2FASetting = ({ organisation }: Force2FASettingProps) => {
+  const [updateOrganisationWithToast, { isLoading: isSaving }] =
+    useUpdateOrganisationWithToast()
+
+  const handleToggle = async () => {
+    await updateOrganisationWithToast(
+      {
+        force_2fa: !organisation.force_2fa,
+        name: organisation.name,
```
Comment (Zaimwa9) [inline frontend/web/components/pages/organisation-settings/tabs/general-tab/sections/admin-settings/Force2FASetting.tsx]: do we need name here ?

File: frontend/web/components/pages/organisation-settings/tabs/general-tab/sections/admin-settings/RestrictProjectCreationSetting.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    useUpdateOrganisationWithToast()
+
+  const hasFeature = Utils.getFlagsmithHasFeature(
+    'restrict_project_create_to_admin',
+  )
+
+  const handleToggle = async () => {
+    await updateOrganisationWithToast(
+      {
+        name: organisation.name,
```
Comment (Zaimwa9) [inline frontend/web/components/pages/organisation-settings/tabs/general-tab/sections/admin-settings/RestrictProjectCreationSetting.tsx]: Same here

File: frontend/web/components/pages/organisation-settings/tabs/general-tab/index.tsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+      <SettingTitle>Organisation Information</SettingTitle>
+      <JSONReference title='Organisation' json={organisation} />
+      <div className='mt-2'>
+        <OrganisationInformation organisation={organisation} />
+        <SettingTitle>Admin Settings</SettingTitle>
+        <Force2FASetting organisation={organisation} />
+        <RestrictProjectCreationSetting organisation={organisation} />
+      </div>
+      <DeleteOrganisation organisation={organisation} />
+    </div>
```
Comment (Zaimwa9) [inline frontend/web/components/pages/organisation-settings/tabs/general-tab/index.tsx]: NIT: Being picky here, I would deal with the 'mt-2' in a parent component `flex-column gap-y-2` and 2 subdivs so the separation of the 2 sub-blocks is crystal clear. But that's really picky I know 😄

**Review by Zaimwa9 — CHANGES_REQUESTED** [review]
So much clearer. Thanks 👍 A couple of comments, mostly minor

File: frontend/web/components/pages/organisation-settings/tabs/general-tab/index.tsx (hunk 6 above)
Comment (talissoncosta) [reply frontend/web/components/pages/organisation-settings/tabs/general-tab/index.tsx]: I totally agree, I always prefer to go with gap instead of margin.

File: frontend/common/services/useOrganisation.ts (hunk 1 above)
Comment (talissoncosta) [reply frontend/common/services/useOrganisation.ts]: Good catch! Changed to use req.id from request params for safer cache invalidation.

File: frontend/common/types/requests.ts (hunk 2 above)
Comment (talissoncosta) [reply frontend/common/types/requests.ts]: Great point. I'd go with `number` based on:
- `RouteContext` (our canonical ID source) provides `organisationId` as `number`
- API responses return organisation IDs as `number` (e.g., `"organisation": 7289`)
- Matches the pattern used for `projectId` in `RouteContext`

File: frontend/web/components/pages/OrganisationSettingsPage.js (hunk 3 above)
Comment (talissoncosta) [reply frontend/web/components/pages/OrganisationSettingsPage.js:79]: You're absolutely right! I've restored the original `/create` fallback behavior. When a user deletes their last organisation, they should be redirected to `/create` to set up a new one immediately, rather than `/organisations` which would just show an empty list.

File: frontend/web/components/pages/organisation-settings/tabs/general-tab/sections/admin-settings/Force2FASetting.tsx (hunk 4 above)
Comment (talissoncosta) [reply frontend/web/components/pages/organisation-settings/tabs/general-tab/sections/admin-settings/Force2FASetting.tsx]: Yes, `name` is required by the API. I've added inline comments (`// name is required by API`) to clarify this.

File: frontend/web/components/pages/organisation-settings/tabs/general-tab/sections/admin-settings/RestrictProjectCreationSetting.tsx (hunk 5 above)
Comment (talissoncosta) [reply frontend/web/components/pages/organisation-settings/tabs/general-tab/sections/admin-settings/RestrictProjectCreationSetting.tsx]: Yes, `name` is required by the API. I've added inline comments (`// name is required by API`) to clarify this.

File: frontend/web/components/pages/organisation-settings/tabs/general-tab/index.tsx (hunk 6 above)
Comment (talissoncosta) [reply frontend/web/components/pages/organisation-settings/tabs/general-tab/index.tsx]: Great suggestion! Restructured the component to use semantic divs with `d-flex flex-column gap-2` for clearer visual separation between the "Organisation Information" and "Admin Settings" sections. Much cleaner now! 👍

File: frontend/web/components/pages/organisation-settings/hooks/useDeleteOrganisationWithToast.ts [hunk 7]
```diff
@@ -1,27 +1,40 @@
...
         await deleteOrganisation({
-          id: String(organisationId),
+          id: organisationId,
         }).unwrap()
+
+        // Calculate next available organisation after deletion
+        const remaining = organisations?.results?.filter(
+          (org) => org.id !== organisationId,
+        )
+        const nextOrgId = remaining?.[0]?.id
```
Comment (Zaimwa9) [inline frontend/web/components/pages/organisation-settings/hooks/useDeleteOrganisationWithToast.ts]:
```suggestion
        const nextOrgId = organisations?.results?.filter(
          (org) => org.id !== organisationId,
        )?.[0]?.id
```

And remove comments wdyt ?

**Review by Zaimwa9 — CHANGES_REQUESTED** [review]
One minor comment otherwise looks good

File: frontend/web/components/pages/organisation-settings/hooks/useDeleteOrganisationWithToast.ts (hunk 7 above)
Comment (talissoncosta) [reply frontend/web/components/pages/organisation-settings/hooks/useDeleteOrganisationWithToast.ts]: Thanks @Zaimwa9 

Yep, agree. 

I will keep only this comment on this file:
        // Refresh OrganisationStore to update navbar and other components
        // that rely on the legacy store

**Review by Zaimwa9 — DISMISSED** [review]

**Comment by talissoncosta** [comment]
@Zaimwa9 
I found some bugs, the name was not being updated on the nav, and the orgs list was not being updated when the org is deleted. I am working on some fixes to it.

File: frontend/web/components/navigation/SelectOrgAndProject.tsx [hunk 8]
```diff
@@ -18,6 +19,12 @@ const SelectOrgAndProject: FC<SelectOrgAndProjectType> = ({
   const isOrganisationSelect = document.location.pathname === '/organisations'
   const isCreateOrganisation = document.location.pathname === '/create'
 
+  const organisationId = AccountStore.getOrganisation()?.id
```
Comment (talissoncosta) [inline frontend/web/components/navigation/SelectOrgAndProject.tsx:22]: I’m not able to use useRouteContext here, so the alternative would be to use the route params.

File: frontend/web/components/pages/organisation-settings/OrganisationSettingsPage.tsx [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+  const { organisationId } = useRouteContext()
+  const paymentsEnabled = Utils.getFlagsmithHasFeature('payments_enabled')
+  const isEnterprise = Utils.isEnterpriseImage()
+
+  const {
+    data: organisation,
+    error,
+    isLoading,
+  } = useGetOrganisationQuery(
+    { id: String(organisationId) },
```
Comment (Zaimwa9) [inline frontend/web/components/pages/organisation-settings/OrganisationSettingsPage.tsx]:
```suggestion
    { id: organisationId ?? 0 },
```

I don't like this 0 fallback but with the skip, the request will never be made. It's a bit the safeguard versus the rules of hooks (conditionally invoked).

Maybe that works for now, but I believe at some point there should be an improvement of ParametrizedRoute (or a wrapper component) that ensures the page has the parameters it needs

File: frontend/web/components/pages/organisation-settings/hooks/useDeleteOrganisationWithToast.ts [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+      try {
+        await deleteOrganisation({
+          id: organisationId,
+        }).unwrap()
+
+        const nextOrgId = organisations?.results?.filter(
+          (org) => org.id !== organisationId,
+        )?.[0]?.id
+
+        AppActions.selectOrganisation(nextOrgId)
```
Comment (Zaimwa9) [inline frontend/web/components/pages/organisation-settings/hooks/useDeleteOrganisationWithToast.ts:30]: Just to explore. I am wondering if we could get rid of these AppActions.
selectOrganisation is basically a wrapper of `setCookie`.
```
  selectOrganisation: (id) => {
    API.setCookie('organisation', `${id}`)
    store.organisation = _.find(store.model.organisations, { id })
    store.changed()
  },
```
Maybe we could extract the function from the store and have it in a hook. Although I do understand this starts to be out of scope but curious to have your opinion.

Ultimately here it could be 
```
const nextOrgId = organisations?.results?.filter(
  (org) => org.id !== organisationId,
)?.[0]?.id

API.setCookie('organisation', `${nextOrgId}`)
```

Or 
```const nextOrgId = organisations?.results?.filter(
  (org) => org.id !== organisationId,
)?.[0]?.id

selectOrganisation(nextOrgId) // from a new hook
```

File: frontend/e2e/tests/organisation-test.ts [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+  await click(byId('organisation-link'))
+  await waitForElementVisible(byId('org-settings-link'))
+  await click(byId('org-settings-link'))
+
+  log('Edit Organisation Name')
+  await waitForElementVisible("[data-test='organisation-name']")
+  await setText("[data-test='organisation-name']", 'Test Organisation')
+  await click('#save-org-btn')
+
+  log('Verify Organisation Name Updated in Breadcrumb')
```
Comment (Zaimwa9) [inline frontend/e2e/tests/organisation-test.ts:32]: Could be worth adding a little sleepTimer here just in case the response is a bit slow

Comment (Zaimwa9) [reply frontend/e2e/tests/organisation-test.ts:32]: and same after each request on which we assert the result wdyt ?

File: frontend/web/components/BreadcrumbSeparator.tsx [hunk 12]
```diff
@@ -199,7 +196,7 @@ const BreadcrumbSeparator: FC<BreadcrumbSeparatorType> = ({
         setOrganisationSearch('')
       }
     }
-  }, [open])
+  }, [open, organisationSearch, projectSearch])
```
Comment (Zaimwa9) [inline frontend/web/components/BreadcrumbSeparator.tsx]: NIT: That's a bit aggressive 😄 

Okk, I re-read, I thought it was part of the search but it's just the search.

I think we can straight get rid of the 2 branches `if projectSearch / if organisationSearch` and just clean up when `!open`

**Review by Zaimwa9 — CHANGES_REQUESTED** [review]
Couple of comments 👍

File: frontend/web/components/pages/organisation-settings/hooks/useDeleteOrganisationWithToast.ts (hunk 10 above)
Comment (talissoncosta) [reply frontend/web/components/pages/organisation-settings/hooks/useDeleteOrganisationWithToast.ts:30]: I totally agree, I'd love to get rid of the store as much as possible.
That is good to know, thanks for the context. I will try those approaches and let you know.

File: frontend/web/components/BreadcrumbSeparator.tsx (hunk 12 above)
Comment (talissoncosta) [reply frontend/web/components/BreadcrumbSeparator.tsx]: It was actually added by lint --fix. I will double check it, thanks for bringing  this up 😃

File: frontend/e2e/tests/organisation-test.ts (hunk 11 above)
Comment (talissoncosta) [reply frontend/e2e/tests/organisation-test.ts:32]: I think as we are already using proper wait helpers (waitForElementVisible, waitForElementNotExist) we are safe on that.

File: frontend/web/components/BreadcrumbSeparator.tsx (hunk 12 above)
Comment (talissoncosta) [reply frontend/web/components/BreadcrumbSeparator.tsx]: Thanks for the suggestion. 
Done! 😃

File: frontend/web/components/pages/organisation-settings/hooks/useDeleteOrganisationWithToast.ts (hunk 10 above)
Comment (talissoncosta) [reply frontend/web/components/pages/organisation-settings/hooks/useDeleteOrganisationWithToast.ts:30]: I don't think we can get rid of it now. BreadcrumbSeparator.tsx (lines 173-188) still relies on AccountStore.on('change') events, which are triggered by AppActions.selectOrganisation() calling store.changed(). We'd need to migrate that component to RTK Query first.

File: frontend/e2e/tests/organisation-test.ts (hunk 11 above)
Comment (Zaimwa9) [reply frontend/e2e/tests/organisation-test.ts:32]: All good then my bad

File: frontend/web/components/pages/organisation-settings/hooks/useDeleteOrganisationWithToast.ts (hunk 10 above)
Comment (Zaimwa9) [reply frontend/web/components/pages/organisation-settings/hooks/useDeleteOrganisationWithToast.ts:30]: Alright! Thanks for checking 🙏

**Review by Zaimwa9 — DISMISSED** [review]
Awesome thanks for the great work, happy to approve

**Comment by talissoncosta** [comment]
@Zaimwa9 
Just one small change, removing unnecessary code.

**Review by Zaimwa9 — APPROVED** [review]
